packages/vue/package.json

```diff
@@ -25,6 +25,9 @@
   "peerDependencies": {
     "vue": "2.x || 3.x"
   },
+  "devDependencies": {
+    "vue": "~3.2.41"
+  },
   "scripts": {
```

Comment:
l: I'm probably missing something obvious but are we using the Vue dependency somewhere? AFICT, we're still importing the `Vue` type from our type definition.

Reply:
we use it in the test now :)

Reply:
Ahh makes sense :D